### ISSUE TITLE
NVDA doesn't save your configurations

### ISSUE THREAD

**Issue by jcoffin1** [nvaccess/nvda#12112]

**Comment by Brian1Gaff** [comment]
What about other bits of a config? I can think of several reasons why it 
might be good to always start with speech on, if nothing else to make sure 
you know its actually working.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by jcoffin1** [comment]
Hi. 

 

I think other configurations are working just fine. I’m using braille with NVDA, so don’t really want speech to run immediate when NVDA starts.

**Comment by lukaszgo1** [comment]
@jcoffin1 If you've turned speech off by pressing NVDA+s the fact that this has not been saved is expected as this shortcut is not permanent. To turn off speech permanently you need to choose 'No speech' in the synthesizer selection dialog.

**Comment by Brian1Gaff** [comment]
That was what I understood as well.
 One of the problems I have at the moment is that due to slow initiation of 
the sound driver on my machine, sometimes nvda says it cannot find a sound 
system to use and sets it to speech silence. This then needs a reboot to get 
it to work. If it did save the speech setting I'd have to uses another copy 
of nvda to get the sound back speaking, but I don't which suits me. As 
suggested if turned off completely this is saved.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by Adriani90** [comment]
I am closing this issue since it seems the issuer has most probably used the temporary shortcut nvda+s to turn off the speech. @jcoffin1 if not so, please comment with clear steps to reproduce and we can reopen the issue.
